projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -2737,6 +2749,30 @@ public static function wrap_message_in_html_tags( $title, $body, $footer, $actio
 		 * @param string the filename of the HTML template used for response emails to the form owner.
 		 */
 		require apply_filters( 'jetpack_forms_response_email_template', __DIR__ . '/templates/email-response.php' );
+
+		$logo_html = '';
+		if ( $has_email_logo ) {
+
+			// Add the logo styles at the end.
+			$style = str_replace( '</style>', '.powered-by a {text-decoration: none;} @media only screen and (max-width: 640px) { .powered-by { padding: 0 16px 16px!important; }} </style>', $style );
```

Comment:
Wondering if it would be clearer/simpler to just leave the styles in the CSS block with the rest of the styles?

It's really minimal so it wouldn't matter much to include, but makes modifying CSS in future easier.